app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html

```diff
@@ -91,14 +91,19 @@
     }
 
     @H1(messages("undertakingSector.title"), Some("govuk-heading-xl govuk-!-margin-bottom-5"))
```

Comment:
I think you were meant to remove the `h1` and the `p` tag and add them as part of the `govukRadios(Radios( ` definition. You would then set `isPageHeading` to true.

Reply:
done